Code/GraphMol/SmilesParse/SmartsWrite.h

```diff
@@ -26,7 +27,11 @@ RDKIT_SMILESPARSE_EXPORT std::string GetBondSmarts(const QueryBond *qbond,
 
 class ROMol;
 //! returns the SMARTS for a molecule
-RDKIT_SMILESPARSE_EXPORT std::string MolToSmarts(ROMol &mol,
+RDKIT_SMILESPARSE_EXPORT std::string MolToSmarts(const ROMol &mol,
+                                                 bool doIsomericSmarts = true);
+
+RDKIT_SMILESPARSE_EXPORT std::string MolFragmentToSmarts(const ROMol &mol,
```

Comment:
Any particular reason you don't also allow the bondsToUse to also be provided (like molFragmentToSmiles)?

Reply:
complete oversight - I didn't actually look at the MolFragmentToSmiles prototype!